tokens.html

```diff
@@ -0,0 +1,335 @@
+
+		<dt><code>comment</code></dt>
+		<dd>Code comments.
+		<pre><code class="language-markup">&lt;!-- Here's a comment --&gt;
+&lt;style&gt;
+	/* Here's another comment */
+&lt;/style&gt;
+&lt;script&gt;
+// Here's yet another comment
+&lt;/script&gt;</code></pre></dd>
+
+		<dt><code>inserted</code> and <code>deleted</code></dt>
+		<dd>Added or modified line and deleted line respectively, mainly for diffs. In general, also the idea of something being increased and decreased/removed respectively.
+		<pre><code class="language-diff">--- qcli.yml	2014-12-16 11:43:41.000000000 +0800
++++ /Users/uranusjr/Desktop/qcli.yml	2014-12-31 11:28:08.000000000 +0800
+@@ -4,5 +4,5 @@
+project:
+	sources: "src/*.cpp"
+	headers: "src/*.h"
+-    qt: core
++    qt: core gui
+public_headers: "src/*.h"</code></pre></dd>
+
+		<dt><code>bold</code></dt>
+		<dd>Bolded text. Mostly found in document-markup languages.
+		<pre><code class="language-markdown">**I am bolded text!**</code></pre></dd>
+
+		<dt><code>italic</code></dt>
+		<dd>Italicised text. Mostly found in document-markup languages.
+		<pre><code class="language-markdown">*I am italicised text!*</code></pre></dd>
+
+		<dt><code>tag</code></dt>
+		<dd>A markup tag (e.g. HTML and XML tags).
+		<pre><code class="language-markup">&lt;p&gt;Hello World!&lt;/p&gt;</code></pre></dd>
+
+		<dt><code>attr-name</code> and <code>attr-value</code></dt>
+		<dd>Kind of like a property of a markup tag and its value/argument respectively.
+		<pre><code class="language-markup">&lt;p id="greeting"&gt;Hello World!&lt;/p&gt;
+&lt;video width="1280" height="720" allowfullscreen controls&gt;
+	&lt;source src="hello_world.mp4" type="video/mp4" /&gt;
+&lt;/video&gt;</code></pre></dd>
+
+		<dt><code>namespace</code></dt>
+		<dd>Used to provide uniquely named elements and attributes in XML documents. Outside of markup languages, it is used to tokenize the package/namespace part of identifiers.
+		<pre><code class="language-markup">&lt;html:p foo:bar="baz" foo:weee&gt;&lt;/html:p&gt;</code></pre>
+		<pre><code class="language-csharp">namespace Foo.Bar {}
+using Foo.Bar;</code></pre>
+		<pre><code class="language-java">class Foo extends foo.bar.Foo {
+	java.util.List<foo.bar.Foo.Bar> bar(foo.bar.Baz bat) throws java.lang.IOException {
+		throw new java.lang.UnsupportedOperationException("Not implemented");
+	}
+}</code></pre>
+		<pre><code class="language-rust">use std::collections::HashMap;
+use std::fmt;
+use std::iter;
+use std::result;
+use std::sync::Arc;
+
+use regex_syntax::hir::{self, Hir};
+use regex_syntax::is_word_byte;
+use regex_syntax::utf8::{Utf8Range, Utf8Sequence, Utf8Sequences};
+
+use crate::prog::{
+		EmptyLook, Inst, InstBytes, InstChar, InstEmptyLook, InstPtr, InstRanges,
+		InstSave, InstSplit, Program,
+};
+
+use crate::Error;
+
+type Result = result::Result<Patch, Error>;
+type ResultOrEmpty = result::Result<Option<Patch>, Error>;</code></pre></dd>
```

Comment:
Let's shorten this a little.

```suggestion
use std::fmt;
use std::sync::Arc;</pre></dd>
```